src/worker/executors/select/index.ts

```diff
@@ -58,7 +59,12 @@ export class Select extends BaseFetch {
             this.limitRecord = query.limit;
         }
         if (query.order) {
-            if (isArray(query.order) || query.order.case || isObject(query.order.by)) {
+            if(isArray(query.order)){
```

Comment:
hey, i am not sure if these are needed. was test cases failing after you changed typing ?

Reply:
Sorry for delayed answer. 
I also think that part is a waste. That part of code is aftermath from fixing problem without type guard function.

And none of test cases were failed after changed typing.

Reply:
fixed and pushed